src/api/app/views/webui/project/_tabs.html.erb

```diff
@@ -14,7 +14,7 @@
     <% else %> <!-- also for incident project -->
       <%= tab 'repositories', "Repositories", :controller => :project, :action => :repositories unless @project.is_remote? or @spider_bot %>
     <% end %>
-    <% if @project.api_obj.repositories && !@spider_bot -%>
+    <% if @project.repositories.any? && !@spider_bot -%>
```

Comment:
It's a relation, can't be nil. It will always be a collectionproxy with or without elements.